ligo/main.mligo

```diff
@@ -17,12 +17,12 @@ let rec initialize_tick ((ticks, i, i_l,
     if Big_map.mem i ticks then
         ticks
     else if i_l.i > i.i then
-        (failwith "Invalid witness" : tick_map)
+        (failwith invalid_witness_err : tick_map)
     else
-        let tick = get_tick ticks i_l in
+        let tick = get_tick ticks i_l tick_not_exist_err in
         let i_next = tick.next in
         if i_next.i > i.i then
-            let tick_next = get_tick ticks i_next in
+            let tick_next = get_tick ticks i_next tick_not_exist_err in
```

Comment:
Here the error is also internal, since `tick.next` is expected to refer to an existing tick.